src/storage/segcache/src/rand.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+pub use inner::*;
+
+pub use rand::Rng as RandRng;
+pub use rand::RngCore as RandRngCore;
+
+// TODO(bmartin): this might be dangerous to gate with the `cfg(test)`. The
```

Comment:
This is probably mitigated now.

Reply:
Leaving this open for now. Worth discussing using PRNG vs CSPRNG

Reply:
To follow-up with some benchmarking. Per discussion, CSPRNG is probably overkill

Reply:
Comments cleaned up